service/src/main/java/bio/terra/profile/service/iam/SamService.java

```diff
@@ -166,6 +163,84 @@ public List<UUID> listProfileIds(AuthenticatedUserRequest userRequest)
     }
   }
 
+  public List<SamPolicyModel> retrieveProfilePolicies(
```

Comment:
Would it be worth it to make more liberal use of javadoc comments generally, but particularly for the primary entrypoints to a class? 

Not thinking anything fancy, but it can make things easier to work with from other classes, since the javadocs will show up in IDEs, as well as a few other knock-on benefits.

Not saying you should do anything about it here, but I figured it was a reasonable time to bring it up.

Reply:
Good call, I will add JavaDoc to the public methods.